chainer/functions/activation/softplus.py

```diff
@@ -58,18 +58,27 @@ def backward_gpu(self, inputs, grads):
 
 
 def softplus(x, beta=1.0):
-    """Elementwise softplus function.
+    """Element-wise softplus function.
 
-    This function is expressed as
-    :math:`f(x) = \\frac{1}{\\beta}\\log(1 + \\exp(\\beta x))`,
-    where :math:`\\beta` is a parameter.
+    .. math:: f(x)=\\frac{1}{\\beta}\\log(1 + \\exp(\\beta x)).
```

Comment:
Please add a notation for `beta`, for example:
where `beta` denotes a parameter.